requirements.txt

```diff
@@ -1,4 +1,4 @@
-altair>=4.0.0,<4.2.1 # conflict with jsonschema
+altair>=4.2.1
```

Comment:
This Altair 4.2.1 includes [this PR](https://github.com/altair-viz/altair/pull/2683), which is currently blocking compatibility with `pandas 2.0.0`